templates/partials/search/results-page.html

```diff
@@ -15,7 +15,7 @@
     <div class="large-3 medium-4 small-6 columns">
       <div class="control control--filters control--active" id="control--filters">
           <label class="control__label" for="fb">Refine search</label>
-          <button class="control__button" id="fb">{{> global/icon i="filter_list" }}</button>
+          <button class="control__button hidden" id="fb">{{> global/icon i="filter_list" }}</button>
```

Comment:
Ok. cool. if you're going to _hide_ it initially and only display if JS is available... 👍